webpack.common.js

```diff
@@ -0,0 +1,66 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const ManifestPlugin = require('webpack-pwa-manifest')
+const { WebpackManifestPlugin } = require('webpack-manifest-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
```

Comment:
Is there a reason why this is w/ `require` as opposed to the ESM `import/export` syntax? AFAIK post node 10 it seems like everybody is moving to ESM. Not super familiar with the webpack use-case though!

Reply:
Good point!! It's a v interesting read if you are interested--basically, [Node.js treats JavaScript code as CommonJS modules by default](https://nodejs.org/api/esm.html#esm_enabling), so [hence the `require`.](https://webpack.js.org/api/module-methods/#import) However, [ECMAScript is preferred in general](https://web.dev/commonjs-larger-bundles/), so I'll switch over!

Reply:
nvm actually, i think almost all webpack tutorials are in commonJS, (and with `require`) and that being the case, i think it might be better to leave as is [(since it uses `__dirname` -- CommonJs variables that are not in ES modules, as per Node.org](https://github.com/nodejs/help/issues/2907#issuecomment-757446568))

I'm open to converting it if converting seems better though!

Reply:
If that's the convention, then fine by me!